jf_agent/jf_jira/jira_download.py

```diff
@@ -60,24 +60,7 @@ def run_jira_download(config: ValidatedConfig, ingest_config: IngestionConfig) -
         MAKARA_CUSTOM_FIELD_MISMATCH_AND_DETECTION_FLAG, False
     ):
         try:
-            ids_to_redownload = detect_and_repair_custom_fields(ingest_config=ingest_config)
```

Comment:
B: This functionality needs to be retained. We only get the issue IDs to redownload at the beginning (whenever we initially generate `IngestionConfig`). Without this functionality, we won't pick up issues that are marked for redownload until the next agent run.